test/rules/no-object-literal-type-assertion/allow-arguments/test.ts.lint

```diff
@@ -0,0 +1,12 @@
+// Allow object literals to be asserted when used as arguments.
```

Comment:
Just to make sure the `allow-arguments` option doesn't completely break the rule in the future, could you copy & paste the existing test cases into this file? They should still cause complaints for non-argument assertions.

Reply:
Of course. Done 👍